IMC_R/EDA_Raymond/contentdevproposal.md

```diff
@@ -1,31 +1,28 @@
-# TEMPLATE
 
-## :fire: Do not edit this file - copy the template and create your own file.
-
-**[Step-By-Step Technical Blog Guide](https://hq.bitproject.org/how-to-write-a-technical-blog/)**
+# EDA for R
 
 ### :pushpin: Step 1
 **TITLE:**    
-{{Insert_title_here}}
+Exploring Movie Streaming Data With R
 
 **TOPIC:**    
-{{Insert_topic_here}}
+Data Analysis with R
 
 **DESCRIPTION (5-7+ sentences):**    
-{{Insert_description_here}}
+Exploratory data analysis, or EDA for short, is an approach to exploring data that can help users obtain basic insights before performing more complex operations. Although the techniques employed by EDA are relatively simple, EDA is a critical component of any statistical analysis. It allows us to understand the underlying structure of the data and evaluate the validity of our assumptions. This blog will help readers understand EDA and employ these techniques in R. It will use the Kaggle movie streaming dataset to illustrate examples of EDA.
 
 ### :pushpin: Step 2
 :family: **TARGET AUDIENCE (3-5+ sentences):**    
-{{Insert_description_here}}
+This blog is targeted towards students who are interested in data analysis and R. It will require relatively little prior knowledge, although readers would benefit from a high school level understanding of statistics, and maybe some familiarity with coding. Experience with R is unnecessary.
 
 ### :pushpin: Step 3
 > Outline your learning/teaching structure: 
 
 **Beginning (2-3+ sentences):**    
-{{Insert_here}}
+The blog will start off with an introduction of EDA. This will include a description of various techniques used for EDA (primarily different types of graphs and tables). Then, I will explain the purpose and significance of EDA.
 
```

Comment:
why are we analyzing movies in the first place? why important insights can be derived from EDA? Include some motivating questions thatll be addresssed later in the code/discussion